.github/workflows/publish_npmjs.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - run: npm ci
       - run: npm run compile:types
-      - run: npm build:component
+      - run: npm run build:component
```

Comment:
I am a bit confused, this change is already in the master branch, is this some github bug or are you somehow merging against some older commit (that looks like master?) 

https://github.com/ethersphere/bee-dashboard/blob/master/.github/workflows/publish_npmjs.yaml

Reply:
hmmm, it is strange indeed

Reply:
I fixed. I think it was based on an older commit